managed-k8s/README.md

```diff
@@ -18,11 +18,38 @@ whereas each worker node has
 
 Provisioning time varies of course but we can expect to take around 10 minutes to spin up the cluster.
 
+## Retrieve kubeconfig
+
+Inside the cluster.tf you can see a defined output. the configured `value` needs
+to match the `gridscale_k8s` resource. In the given example, our `gridscale_k8s`
+resource is named `mycluster`:
+
+```
+resource "gridscale_k8s" "mycluster" {
+  # ...
+}
+```
+
+So the configured value of our terraform `output` needs to look like this:
+
+```
+output "kubeconfig" {
+  value = gridscale_k8s.mycluster.kubeconfig
+}
+```
+
+When you create your cluster terraform will write the kubeconfig now to STDOUT,
```

Comment:
Can you make this lowercase "stdout" instead of STDOUT?

Reply:
done